extensions/ql-vscode/test/matchers/toEqualPath.ts

```diff
@@ -0,0 +1,53 @@
+import { expect } from "@jest/globals";
+import type { MatcherFunction } from "expect";
+import { pathsEqual } from "../../src/pure/files";
+
+// eslint-disable-next-line func-style -- We need to have access to this and specify the type of the function
+const toEqualPath: MatcherFunction<[expectedPath: unknown]> = function (
```

Comment:
Is all this so we can do `toEqualPath` in test assertions or have I missed the point?

Reply:
Yes, this is just so there's a wrapper around `pathsEqual`. I agree that it's a lot of code, but otherwise the failed assertion messages are something like "expected true to be false" which is really unhelpful when debugging these tests. Now, it will actually mention the two values and let you compare the values without needing to modify the tests.

Reply:
Ok thanks.